README.md

```diff
@@ -1 +1,21 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+Hi Everyone! I'm Ntazama Eloi and this my portfolio
+
+# Built With
+
+- HTML
+- CSS
+
+
```

Comment:
- [ ] [OPTIONAL] Please write a step-by-step about how to clone the repo and start the project. This will make your project more professional. Remember that not every person that will see your project knows how to program so write the step as much descriptive as you can. 🥇 💪

Reply:
Thank you I will take that into consideration